ghost/admin/app/components/gh-tags-token-input.hbs

```diff
@@ -9,9 +9,9 @@
     @optionsComponent={{component "power-select-vertical-collection-options" lastReached=(perform this.loadMoreTagsTask)}}
     @registerAPI={{this.registerPowerSelectAPI}}
     @renderInPlace={{@renderInPlace}}
-    @search={{perform this.searchTagsTask}}
+    @search={{if this.useServerSideSearch (perform this.searchTagsTask) null}}
```

Comment:
Providing `null` here means `<GhTokenInput>` falls back to it's client-side filtering